src/main/scala/tilelink/Broadcast.scala

```diff
@@ -391,45 +559,81 @@ object TLBroadcast
 }
 
 class ProbeTrackInfo(val caches: Int) extends Bundle {
+  /** how many clients left need to be probed. */
   val count   = UInt(log2Ceil(caches+1).W)
+  /** probe which client now. */
   val cacheOH = UInt(caches.W)
 }
 
+/** Tracker to track trans */
 class TLBroadcastTracker(id: Int, lineBytes: Int, caches: Int, bufferless: Boolean, edgeIn: TLEdgeIn, edgeOut: TLEdgeOut) extends Module
 {
   val io = IO(new Bundle {
-    val in_a_first = Input(Bool())
-    val in_a  = Flipped(Decoupled(new TLBundleA(edgeIn.bundle)))
+    /** first packet come from a. */
+    val in_a_first: Bool = Input(Bool())
+    /** incoming A channel from clients. */
+    val in_a: DecoupledIO[TLBundleA] = Flipped(Decoupled(new TLBundleA(edgeIn.bundle)))
+    /** output A channel to mangers. */
```

Comment:
I'd say "inwards" and "outwards" here to match what we wrote for the diplomacy node docs